docs/Utils.md

```diff
@@ -6,9 +6,10 @@ Some powerfull utils that can help you build best products.
 
 **UtilsService**
 * [.currencyConvert(params)](#currency)
+* [.referenceDataReq(params)](#refdata)
```

Comment:
The name could probably be simplified to `referenceData` as it's obvious that this will be a req :)

Reply:
Sure
This is done and pushed